groups/alteryx-sandbox/iam.tf

```
@@ -30,7 +30,16 @@ module "alteryx_server_profile" {
       actions = [
         "cloudwatch:PutMetricData"
       ]
-    }
+    },
+    {
+      sid     = "S3AllowListGet"
+      effect  = "Allow"
+      resources = ["${data.aws_s3_bucket.resources.arn}/"]
```

The `ListBucket` action applies to bucket resources, and `GetObject` applies to object resources, so you'll need to represent both resources in the policy. Either separate these actions individual separate statements, with one targeting the bucket resource and the other objects within the bucket, or include both resources here:

```suggestion
      resources = [
        "${data.aws_s3_bucket.resources.arn}",
        "${data.aws_s3_bucket.resources.arn}/*"
      ]
```

See [Actions, resources, and condition keys for Amazon S3](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazons3.html) for more information.